saleor/graphql/page/decorators.py

```diff
@@ -0,0 +1,23 @@
+from functools import wraps
+
+from graphql_jwt.exceptions import PermissionDenied
+
+from ...page.models import Page
+
+
+def must_be_unprotected(fn):
```

Comment:
Again, why the decorator? Ideally, this simple condition could be checked inside overwritten `delete_instance` method.